tests/PipelineMicroserviceCLI/Commands/CommandTest.php

```diff
@@ -7,41 +7,89 @@
 
 class CommandTest extends CommandTestCase {
     
-    public function testPublishPipelineMissingBaseUrl(){
+    public function testPublishPipeline(){
         $mockHandler = $this->getHttpMockHandler(["HiddenPipelines.txt", "PublishedPipeline.txt"]);
         $application = new Application();
```

Comment:
It might be interesting to create a custom application class that extends from this one and transparently loads in the configuration depending on the environment.  It would also load in the commands inside so that configuration does not have to be duplicated both in our application front entry and the testing setup.